### DISCUSSION TITLE
Show 24h chance price % in front of each asset on main window

### DISCUSSION THREAD

**Discussion by bitcoinuser** [defray-labs/sovryn-mobile#62] [Ideas]
Hi,

I would like to see the 24h change price % in front of each asset on main window.

**Comment by harmaus** [comment]
Was thinking about it, but my plan is to not depend on any backends with exception to blockchain nodes at this moment. 
At the moment I see two options here:
- have background job which would ping smart-contracts for prices (cons: not reliable using react-native, especially if user kills an app)
- use backend service like Covalent (pros: it's free and without rate limiter; cons: once they will introduce pricing plans, I may get f*cked. Would prefer to wait for paid plans before deciding to depend too heavily on them)

In the end, it would be nice if user would be able to choose pricing provider from different sources somwhere in the app settings.

I'm other to discussions for better approaches, actually converting this issue to discussion until it's ready for development.
